src/app/tools/point_shapes.h

```diff
@@ -131,6 +131,7 @@ class FloodFillPointShape : public PointShape {
 
 class SprayPointShape : public PointShape {
   BrushPointShape m_subPointShape;
+  float point_remainder = 0;
```

Comment:
Rename to "m_pointRemainder" (similar to "m_subPointShape", we use "m_" prefix for member variables)